src/github/pullRequestModel.ts

```diff
@@ -721,6 +723,43 @@ export class PullRequestModel extends IssueModel implements IPullRequestModel {
 		};
 	}
 
+	async openDiffFromComment(comment: IComment): Promise<void> {
```

Comment:
I'm probably missing something here, but all of the properties/methods on `this` in this method seem to be public. To avoid giving `PullRequestModel` a reference the folder repository manager could you make this method static or move it into a utility funciton?

Reply:
Good point! I made this method static and it really reduced this change set. thanks!